pubspec.yaml

```
@@ -1,10 +1,10 @@
 name: platform
-version: 3.0.0-nullsafety
+version: 3.0.0-nullsafety.1
 description: A pluggable, mockable platform abstraction for Dart.
 homepage: https://github.com/google/platform.dart
 
 dev_dependencies:
-  test: ^1.0.0
+  test: ^1.16.0-nullsafety.1
 
 environment:
-  sdk: '>=2.9.0 <3.0.0'
+  sdk: '>=2.10.0-4.0.dev <3.0.0'
```

```suggestion
  sdk: '>=2.10.0-4.0.dev <2.10.0'
```

The upper bound needs to be more restrictive - the language version for nnbd is not finalized yet (and the feature may have breaking changes)